README.md

```diff
@@ -29,7 +29,7 @@ When not using [`phpstan/extension-installer`](https://github.com/phpstan/extens
 
 ```diff
  includes:
-+	- vendor/jangregor/phpstan-prophecy/extension.neon
++	- vendor/jangregor/phpstan-prophecy/src/extension.neon
```

Comment:
@dantleech 

That is actually not the correct path - it's only a symbolic link (for legacy reasons).

Reply:
hmm - the old one is the symbolic link?

the previous configuration didn't work for me.

Reply:
@dantleech

Yes, it should: https://github.com/Jan0707/phpstan-prophecy/blob/0.7.0/src/extension.neon

Does it not?

Reply:
@dantleech

Ha, sorry - that change is only included on [`0.7.0`](https://github.com/Jan0707/phpstan-prophecy/releases/tag/0.7.0), see https://github.com/Jan0707/phpstan-prophecy/commit/5fc5e3c8ae693f7fdc78bb1674a417c2874890cd#diff-300c6013fb9bc1bc02b8fbd9fb9ac2b0.